assets/stylesheets/main.css

```diff
@@ -235,20 +269,27 @@ a:hover#join {
 */
 
 #introduction {
-    display: flex;
+    display: block;
```

Comment:
sections must have a value of "block" for the "display:" property in order for z-index to work.